src/portfft/common/logging.hpp

```diff
@@ -261,6 +261,43 @@ struct global_data_struct {
   }
 };
 
+/**
+ * Prints the message and dumps data from host to standard output
+ * 
+ * @tparam T type of element to dump
+ * @param msg message to print
+ * @param dev_ptr USM pointer to data on device
+ * @param size number of elements to dump
+ */
+template<typename T>
+PORTFFT_INLINE void dump_host([[maybe_unused]] const char* msg, [[maybe_unused]] T* host_ptr, [[maybe_unused]] std::size_t size){
```

Comment:
It would be nice to have an overload that takes a `std::vector` instead of a pointer + size

Reply:
I will do that when I work on logging.